test/Java8andUp/build.xml

```diff
@@ -127,57 +127,57 @@
-					</else>
-				</if>
 				<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
 					<src path="${src}" />
-					<src path="${src_90}" />
+					<src path="${src_80}" />
 					<src path="${TestUtilities}" />
 					<src path="${transformerListener}" />
-					<compilerarg line='${addExports}' />
-					<compilerarg line='${addModules}' />
-					<!-- PR117298-->
 					<exclude name="**/Cmvc194280.java" />
 					<exclude name="**/resources/**"/>
-					<!-- need to be included when OpenJ9 have them. -->
 					<exclude name="**/gpu/**"/>
 					<!-- requires special compilation methods -->
 					<exclude name="**/NoSuchMethod/**" />
 					<classpath>
 						<pathelement location="../TestConfig/lib/asm-all.jar" />
 						<pathelement location="../TestConfig/lib/testng.jar" />
 						<pathelement location="../TestConfig/lib/jcommander.jar" />
+						<pathelement location="${JAVA_BIN}/../../lib/tools.jar" />
 						<pathelement location="../TestConfig/lib/commons-cli.jar" />
 						<pathelement location="../TestConfig/lib/javassist.jar" />
 					</classpath>
 				</javac>
 			</then>
 			<else>
+				<property name="addModules" value="--add-modules java.se.ee,openj9.sharedclasses" />
+				<if>
+					<equals arg1="${JCL_VERSION}" arg2="latest"/>
+					<then>
+						<property name="addExports" value="--add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED --add-exports java.base/com.ibm.oti.vm=ALL-UNNAMED --add-exports java.base/com.ibm.oti.util=ALL-UNNAMED --add-exports java.base/jdk.internal.vm.annotation=ALL-UNNAMED --add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.base/com.ibm.jit.crypto=ALL-UNNAMED --add-exports java.base/com.ibm.jit=ALL-UNNAMED --add-exports java.base/com.ibm.oti.reflect=ALL-UNNAMED --add-exports java.xml.bind/javax.xml.bind.annotation=ALL-UNNAMED --add-exports java.xml.bind/javax.xml.bind=ALL-UNNAMED" />
+					</then>
+					<else>
+						<property name="addExports" value="--add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED --add-exports java.base/com.ibm.oti.vm=ALL-UNNAMED --add-exports java.base/com.ibm.oti.util=ALL-UNNAMED --add-exports java.base/jdk.internal.vm.annotation=ALL-UNNAMED --add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.base/com.ibm.jit.crypto=ALL-UNNAMED --add-exports java.base/com.ibm.jit=ALL-UNNAMED --add-exports java.base/com.ibm.oti.reflect=ALL-UNNAMED --add-exports java.xml.bind/javax.xml.bind.annotation=ALL-UNNAMED --add-exports java.xml.bind/javax.xml.bind=ALL-UNNAMED" />
+					</else>
+				</if>
```

Comment:
are `addExports` in if and else the same?

Reply:
They're the same here. I'll remove the unneeded if statement.